example/lib/main_page.dart

```diff
@@ -138,13 +138,13 @@ class _MyHomePageState extends State<MyHomePage> {
             ),
             Text(
               '$_counter',
-              style: Theme.of(context).textTheme.display1,
+              style: Theme.of(context).textTheme.headline3,
             ),
-            FlatButton(
+            TextButton(
               child: Text('Click here to log error'),
               onPressed: _clickLogErrorButton,
             ),
-            RaisedButton(
+            ElevatedButton(
               child: Text('Check SDK state'),
               onPressed: _checkSdkState,
             )
```

Comment:
**Suggestion:** The onPressed function for the ElevatedButton and TextButton could be made null-safe to avoid potential null pointer exceptions.
```suggestion
            TextButton(
              child: Text('Click here to log error'),
              onPressed: _clickLogErrorButton ?? () {},
            ),
            ElevatedButton(
              child: Text('Check SDK state'),
              onPressed: _checkSdkState ?? () {},
            )
```